packages/x-tailwindcss/src/x-tailwind-plugin/components/layout/container.ts

```diff
@@ -1,9 +1,14 @@
+import { TailwindHelpers } from '../../../types';
+
 /**
- * Returns the `default` element CSS.
+ * Returns the `container` element CSS.
  *
+ * @param helpers - The {@link TailwindHelpers} to generate CSS.
  * @returns The {@link CssStyleOptions} for the component.
  */
-export function container() {
+// eslint-disable-next-line max-len
+// eslint-disable-next-line @typescript-eslint/explicit-function-return-type,@typescript-eslint/no-unused-vars
```

Comment:
**suggestion:** remember that we set in the `.eslintrc.js` file the `@typescript-eslint/explicit-function-return-type: off` so we don't have to write it everywhere.

```suggestion
// eslint-disable-next-line @typescript-eslint/no-unused-vars
```